Subject: [Patch] Get correct OpenSSL version from deps
Index: CMakeLists.txt
<+>UTF-8
===================================================================
diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	(revision 2f33be817cbce6ad7a36f27dd7ada9219f13584c)
+++ b/CMakeLists.txt	(date 1698399944886)
@@ -384,7 +384,9 @@
 endif()

 if(CMAKE_USE_OPENSSL)
-  find_package(OpenSSL REQUIRED)
+  set(ENV{PKG_CONFIG_PATH} "${DESTDIR}/usr/local")
+  find_package(OpenSSL 1...<2 REQUIRED)
+  set(ENV{PKG_CONFIG_PATH} "")
   set(SSL_ENABLED ON)
   set(USE_OPENSSL ON)

